easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -21,6 +21,24 @@ builddependencies = [
 dependencies = [
     ('Python', '3.7.4'),
     ('h5py', '2.10.0', versionsuffix),
+    ('cURL', '7.66.0'),
+    ('double-conversion', '3.1.4'),
+    ('flatbuffers', '1.12.0'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.0.3'),
+    ('ICU', '65.1'),
```

Comment:
You should stick to `ICU` 64.2 here to make the tests pass: they're complaining that you're introducing a 2nd dep variant for ICU in the `2019b` generation of easyconfigs, which would make this module incompatible with other modules also loading `ICU`.